src/structure/structure.ts

```diff
@@ -0,0 +1,27 @@
+import { Node, PaginatedNodes } from './dto/paginated-nodes-dto';
+import { HttpClientProxy } from '../utils/http-client-proxy';
+
+export interface Options {
+  filter?: string;
```

Comment:
@JERILJOSE96 it's inconsistent with other interfaces definitions - you use `readonly` in all other places. Please go through your code (file by file) and correct all things that look inconsistent.